lib/widgets/buildTasks.dart

```diff
@@ -61,138 +62,161 @@ class _TasksBuilderState extends State<TasksBuilder> {
-                                    child: const Text('No'),
-                                  ),
-                                  TextButton(
-                                    onPressed: () {
-                                      Navigator.of(context).pop();
-                                    },
-                                    child: const Text('Cancel'),
-                                  ),
-                                ],
-                              );
-                            },
-                          );
-                        },
-                        icon: Icons.done,
-                        label: "COMPLETE",
-                        backgroundColor: Colors.green,
-                      ),
-                    ],
-                  ),
-                  endActionPane: ActionPane(
-                    motion: const DrawerMotion(),
-                    children: [
-                      SlidableAction(
-                        onPressed: (context) {
-                          showDialog(
-                            context: context,
-                            builder: (context) {
-                              return AlertDialog(
-                                title:
-                                    const Text('Do you want to save changes?'),
-                                actions: [
-                                  TextButton(
-                                    onPressed: () {
-                                      setStatus('deleted', task.uuid);
-                                      Navigator.of(context).pop();
-                                    },
-                                    child: const Text('Yes'),
-                                  ),
-                                  TextButton(
-                                    onPressed: () {
-                                      Navigator.of(context).pop();
-                                    },
-                                    child: const Text('No'),
-                                  ),
-                                  TextButton(
-                                    onPressed: () {
-                                      Navigator.of(context).pop();
-                                    },
-                                    child: const Text('Cancel'),
-                                  ),
-                                ],
-                              );
-                            },
-                          );
-                        },
-                        icon: Icons.delete,
-                        label: "DELETE",
-                        backgroundColor: Colors.red,
-                      ),
-                    ],
-                  ),
-                  child: Card(
-                    color:
-                        AppSettings.isDarkMode ? Palette.kToDark : Colors.white,
-                    child: InkWell(
-                      customBorder: RoundedRectangleBorder(
```

Comment:
Hey, maybe you have removed this part of the code which is fixed for this issue https://github.com/CCExtractor/taskwarrior-flutter/issues/65

 "customBorder: RoundedRectangleBorder(
                      borderRadius: BorderRadius.circular(10),
                    ),"